@ornikar/eslint-config-typescript/rules/typescript-eslint.js

```diff
@@ -1,13 +1,7 @@
 'use strict';
 
 module.exports = {
-  parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint'],
-  extends: [
-    'plugin:@typescript-eslint/eslint-recommended',
-    'plugin:@typescript-eslint/recommended',
-    'plugin:@typescript-eslint/recommended-requiring-type-checking',
-  ],
+  extends: ['plugin:@typescript-eslint/recommended', 'plugin:@typescript-eslint/recommended-requiring-type-checking'],
```

Comment:
ℹ️ parser and plugins are already imported in recommended config.  `eslint-recommended` is also imported in recommended config.